recipes/gtest/all/conanfile.py

```diff
@@ -97,12 +120,14 @@ def _configure_cmake(self):
 
     def build(self):
         for patch in self.conan_data["patches"][self.version]:
-            tools.patch(**patch)
+            tools.patch(patch)
         cmake = self._configure_cmake()
         cmake.build()
+        if self.options.gtest_build_tests or self.options.gmock_build_tests:
+            self.run(f"ctest")
```

Comment:
```suggestion
```

Do not run unit tests here. The idea for CCI packages is packaging only.

Reply:
We would like the run the unit test continuously to test if the conan patches do not change the behaviour of gtest. If you enable the options the unit-test will be automatically executed, this would fit the mindset of continuous integration.